### PR TITLE
fix(EMailTemplate): malformed HTML in some cases

### PR DESCRIPTION
**Pull request by tomaski**
when an addBodyListItem() is not directly followed by addBodyButton(), resulting HTML is broken

### CODE REVIEW

File: lib/private/Mail/EMailTemplate.php [hunk 1]
```diff
@@ -436,6 +436,7 @@ public function addBodyText(string $text, $plainText = '') {
 		}
 
 		$this->ensureBodyIsOpened();
+		$this->ensureBodyListClosed();
```
Comment (nickvergessen) [inline lib/private/Mail/EMailTemplate.php]: the order should be the otherway around:
```
		$this->ensureBodyListClosed();
		$this->ensureBodyIsOpened();
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: lib/private/Mail/EMailTemplate.php (hunk 1 above)
Comment (tomaski) [reply lib/private/Mail/EMailTemplate.php]: thx @nickvergessen. fixed now

**Review by nickvergessen — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]
Code makes sense 👍

**Pull request merged**